tutorials/simulation/10_array_objs.py

```diff
@@ -212,9 +212,8 @@
 # decomposition for estimation of power spectra. Or you may wish to
 # process pre-computed power spectra in MNE.
 # Following the same logic, it is possible to instantiate averaged power
-# spectrum using the :class:`~mne.time_frequency.Spectrum` class.
-# This is slightly
-# experimental at the moment but works. An API for doing this may follow.
+# spectrum using the :class:`~mne.time_frequency.SpectrumArray` class.
+# This is slightly experimental at the moment but works.
```

Comment:
Let's assume that by the end of this PR it won't be experimental any more.  Otherwise we'll probably forget to update this before release.

```suggestion
```